taqueria-protocol-types/out/types/ConfigArtifactsDir.ts

```diff
@@ -0,0 +1,48 @@
+// Generated file: Do not edit
```

Comment:
We shouldn't track anything within the `/out` folder right? Or anything generated

Reply:
right - this was added to gitignore

Reply:
@ricklove And we probably need another commit to untrack it too right?